src/services/connections.js

```diff
@@ -549,7 +569,11 @@ module.exports = class ConnectionHelper {
 			}
 
 			// Get email template
-			const templateData = await notificationQueries.findOneEmailTemplate(templateCode, orgId.toString())
+			const templateData = await notificationQueries.findOneEmailTemplate(
+				templateCode,
+				orgId.toString(),
+				tenantCode
```

Comment:
@suman how you are adding this with out getting tenantCode from the previous function ?